src/js/components/Grid/stories/typescript/App.tsx

```diff
@@ -58,12 +59,17 @@ export const AppGrid = () => {
       ) : (
         <Text>Grid is not supported by your browser</Text>
       )}
-    </Grommet>
+    </Box>
+    // </Grommet>
   );
 };
 
 AppGrid.storyName = 'App';
 
+AppGrid.args = {
```

Comment:
```suggestion
App.args = {
```
I think this might just need to be 'App' since we change the story name to App in the line above. Currently the story isn't displaying with the full prop set

Reply:
So I could not change to App in this line, because App is not defined. I moved above this command line and confirmed in storybook, height is 100vh and width 100vw(it was before change too).
Or maybe I didn't get the problem, in this case, could you explain with more details the problem? So I can fix again :).

Reply:
Thanks for testing this out. I just tested locally and I think we can leave this line as `AppGrid.args = {` and instead remove the `<Box>` that is wrapping the story and just replace with `<>`